### PR TITLE
Fix couple of UX items

### PR DESCRIPTION
**Pull request by BryanEliDimas**
https://github.com/openstax/business-intel/issues/918

### CODE REVIEW

File: app/views/newflow/educator_signup/educator_change_signup_email_form.html.erb [hunk 1]
```diff
@@ -1,8 +1,17 @@
+<%
+    current_step = I18n.t(
+        :"login_signup_form.step_counter",
+        current_step: 2,
+        total_steps: @total_steps
+    )
```
Comment (steveburkett) [inline app/views/newflow/educator_signup/educator_change_signup_email_form.html.erb:6]: the current step is another one of those pieces of state data that would be good to expose to the views thru a data object as a PORO.   fine i guess for here...but just sayin

File: app/controllers/newflow/educator_signup_controller.rb [hunk 2]
```diff
@@ -33,6 +33,7 @@ def educator_signup
 
     def educator_change_signup_email_form
       @email = unverified_user.email_addresses.first.value
+      @total_steps = 4
```
Comment (steveburkett) [inline app/controllers/newflow/educator_signup_controller.rb:36]: personally, i really dislike "magic numbers" (better to put all this state stuff into a PORO that's passed as a component of a view-controller interface adapter object.  but that would be a lot more work.

File: app/views/newflow/educator_signup/educator_change_signup_email_form.html.erb (hunk 1 above)
Comment (BryanEliDimas) [reply app/views/newflow/educator_signup/educator_change_signup_email_form.html.erb:6]: Right; every single variable in the view(s) between `<%` and `%>` CAN be eventually put into a ViewModel — as part of the "Model–view–viewmodel" concept.

At this time, the whole Accounts app is an MVC though. Future improvement for sure 👍 .

File: app/controllers/newflow/educator_signup_controller.rb (hunk 2 above)
Comment (BryanEliDimas) [reply app/controllers/newflow/educator_signup_controller.rb:36]: Would it be worse or better if there was no instance variable for the number 4. I feel like I went one (small) step beyond just putting it in the view—by putting it as an instance variable. We're still (as a team) not totally decided if we are ever going to skip the entire step 3 or... not, that's still in the works but yeah, for now I think that it doesn't hurt to keep this instance variable—for now.

Comment (BryanEliDimas) [reply app/controllers/newflow/educator_signup_controller.rb:36]: Again, for the record, I also dislike "magic numbers" and I agree about creating a PORO (a ModelView object), but those are small improvements I hope/plan to do soon in a future PR.

**Pull request merged**